examples/htmlStripper.py

```diff
@@ -17,7 +16,7 @@
 
 # get some HTML
 targetURL = "https://wiki.python.org/moin/PythonDecoratorLibrary"
-with closing(urllib.request.urlopen( targetURL )) as targetPage:
+with urllib.request.urlopen( targetURL ) as targetPage:
```

Comment:
Can we have `urlopen(...)` everywhere instead of the verbose `urllib.request.urlopen(...)`

Reply:
Yup! Can do.